README.md

```diff
@@ -77,3 +79,11 @@ If you'd like to get the information from Cartographer to go into Ardupilot's ex
 The parameters above are recommended for SITL. If you plan on using this on a real copter, it is a good idea to setup a second source of EKF. This way the robot doesn't crash if the external odometry you are providing stops publishing or gets lost.
 
 Please refer to this link for more information on [Common EKF Sources](https://ardupilot.org/copter/docs/common-ekf-sources.html>) as well as this guide on [GPS / Non-GPS Transitions](https://ardupilot.org/copter/docs/common-non-gps-to-gps.html).
+
+## Contribution Guideline
+* Ensure the pre-commit hooks pass locally before creating your pull request by installing the hooks before committing.
```

Comment:
Looks good. Only suggestion would be to add a link to the pre-commit user guide: https://pre-commit.com/

Reply:
Done.